recipes-support/salt/files/run-ptest

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+# Kill any salt test processes which may be left around from orphaned test
+# runs.
+killall -q cli_master.py
+killall -q cli_syndic.py
+killall -q cli_minion.py
+killall -q cli_run.py
+
+python PYTHON_SITEPACKAGES_DIR/salt-tests/tests/runtests.py \
```

Comment:
Does python let you do something like `python -m salt-tests.tests.runtests` and do the site-package locating for you?

Reply:
Yes, that works. I'll replace this line with that.